test/com/facebook/buck/android/AndroidBinaryIntegrationTest.java

```diff
@@ -16,6 +16,7 @@
 
 package com.facebook.buck.android;
 
+import static com.facebook.buck.testutil.RegexMatcher.containsRegex;
```

Comment:
I think you need to add a dep for this.  `buck build //test/com/facebook/buck/android:` should reproduce a build failure here.